2 - UI KIT/Apple Pie/Apple Pie/ViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  ViewController.swift
+//  Apple Pie
+//
+//  Created by Student07 on 2019-04-16.
+//  Copyright © 2019 Student07. All rights reserved.
+//
+
+import UIKit
```

Comment:
Small comment and nothing critical but you'll find as you write more that having some order/organization to the file will make it easier to read. I'd suggest the following sections, with a line of whitespace or a `//MARK: - ` between them:
* @IBOutlets
* @IBActions
* let/var properties
* view controller lifecycle funcs - viewDidLoad, viewWillAppear
* the rest